core/src/test/scala/org/apache/spark/scheduler/SparkListenerSuite.scala

```diff
@@ -142,6 +142,7 @@ class SparkListenerSuite extends SparkFunSuite with LocalSparkContext with Match
     stageInfo.submissionTime should be ('defined)
     stageInfo.completionTime should be ('defined)
     taskInfoMetrics.length should be {4}
+    sc.stop()
```

Comment:
`LocalSparkContext` already manages this, so is the problem that these tests should not be making their own context to begin with?